pkg/corroborate/testdata/bigdemo/gen.py

```
@@ -0,0 +1,242 @@
+K8S_POOL = ["1.33", "1.32", "1.31", "1.30"]
+
+
+def H(*parts):
+    return int(hashlib.sha256("|".join(map(str, parts)).encode()).hexdigest(), 16)
+
+
+# --- signer roster (class + allowlisted are pre-derived by GP2 at ingest) ----
+SIGNERS = {
+    "nvidia":    dict(label="NVIDIA UAT",   cls="first-party", allow=True,
+                      identity="https://github.com/NVIDIA/aicr/.github/workflows/uat.yaml@refs/heads/main",
+                      issuer="https://token.actions.githubusercontent.com"),
```

Ruff (0.15.18)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)